monitors/filterWords.js

```diff
@@ -0,0 +1,34 @@
+exports.run = async (client, msg) => {
+  if (!msg.guild) return;
+  const { modLog, swearWords } = msg.guild.settings;
+  if (!msg.deletable && !swearWords.length > 0) return;
+  const badWords = swearWords.filter(word => msg.content.toLowerCase().includes(word));
+  if (badWords.length > 0) {
+    await msg.author.send(`I deleted the message below because you used a word that is not allowed. : **${badWords.join(", ")}**`);
+    await msg.author.send(msg.content);
+    msg.delete().then(m => m.guild.channels.get(modLog).send(`Deleted message from ${m.author.tag} that said the \n\n **Naughty Words:** ${badWords} in the server ${m.guild.name} in the channel ${m.channel.name}. \n\n **Full Message:** ${m.content}`))
```

Comment:
You forgot a return. And put the `then` in the next line. Also, you're not checking the existence of the modLog key (it could be null and error).